src/main/java/gregtech/api/recipes/logic/ParallelLogic.java

```diff
@@ -486,7 +486,7 @@ public static RecipeBuilder<?> appendItemRecipes(RecipeMap<?> recipeMap, IItemHa
             if (matchingRecipe != null) {
                 inputIngredient = matchingRecipe.getInputs().get(0);
                 if (recipeBuilder == null) {
-                    recipeBuilder = recipeMap.recipeBuilder();
+                    recipeBuilder = recipeMap.recipeBuilder().EUt(0);
```

Comment:
Does this also need to be applied to duration? Some recipe Maps, like the macerator, auto set both a duration and an EUt value if it is not specified?